src/openms/source/CHEMISTRY/RibonucleotideDB.cpp

```diff
@@ -37,15 +37,36 @@
 #include <OpenMS/SYSTEM/File.h>
 #include <QFile>
 #include <QTextStream>
+#include <nlohmann/json.hpp>
+
 
 using namespace std;
 
 namespace OpenMS
 {
+
+  struct ParsedEntry_
```

Comment:
please add some documentation on the struct and data members
e.g. it should already be obvious why there are exactly two alternatives etc.